### ISSUE TITLE
height, verticalAlignment doesn't work

### ISSUE THREAD

**Issue by TChengZ** [BohemianCoding/SketchAPI#447]
i create a textLayer in such code which set stringValue, verticalAlignment, frame and so on in sketch 52.6
```js
var Text = require('sketch/dom').Text;
var layerObjectJS = new Text({
            parent: currentLayer.parentArtboard(),
            text: "123",
            name: "123",
            fixedWidth: 1,
           alignment:2,
           frame:{x: 10,  y:10, width:145, height:145}
           });
var layerObject = layerObjectJS._object;
layerObject.textColor = MSImmutableColor.colorWithSVGString("#0000FF");

        //set paragraph
            var paragraphStyle = layerObject.paragraphStyle();
            paragraphStyle.setParagraphSpacing("3");
            layerObject.addAttribute_value("NSParagraphStyle", paragraphStyle);

        //set font
        var newFont = NSFont.fontWithName_size("PingFangSC-Regular", 30);
        layerObject.setFont(newFont);

        layerObject.setKerning(2);
      layerObject.verticalAlignment = 2;
```
since i want make the text height of 90 but it result to 42 which just wrap the text
![image](https://user-images.githubusercontent.com/2180654/55456923-e903f580-561a-11e9-8b20-79fa130fb08e.png)

the i use
```js
currentLayer.absoluteRect().height = 90;
```
the height of layer is 90 but the verticalAlignment is wrong, and can't be edit otherwise you change the height of layer manual not by code
![image](https://user-images.githubusercontent.com/2180654/55457247-fec5ea80-561b-11e9-9221-b859893c7869.png)

it seem that when the height of layer is bigger than the text, verticalAlignment can not work

**Comment by TChengZ** [comment]
Any idea about this?

**Comment by KevinGutowski** [comment]
I can reproduce as well (on Sketch 54). Something seems wrong when attempting to set a frame and adding any sort of styling.

```
let sketch = require('sketch')
let Text = require('sketch').Text
let Rectangle = require('sketch/dom').Rectangle

let document = sketch.getSelectedDocument()
let page = document.selectedPage

let textLayer = new Text({
  parent: page,
  text: "123",
  name: "textLayer",
  frame: new Rectangle(10,10,125,125),
})
```

Works as expected but as soon as you attempt to style it the frame gets updated to a different location:
```
let sketch = require('sketch')
let Text = require('sketch').Text
let Rectangle = require('sketch/dom').Rectangle

let document = sketch.getSelectedDocument()
let page = document.selectedPage

let textLayer = new Text({
  parent: page,
  text: "123",
  name: "textLayer",
  frame: new Rectangle(10,10,125,125),
  style: {}
})
```

I thought that this was fixed with #435 🤔

**Comment by TChengZ** [comment]
i found the solution, in order to make verticalAligment work, the textLayer must setTextBehavior
```
layerObject setTextBehaviour(2);
```

then the height and verticalAlignment all be right
 so hard to find out about it
